R/round.r

```diff
@@ -1,7 +1,7 @@
 #' Round, floor and ceiling methods for date-time objects.
 #'
 #' Users can specify whether to round to the nearest second, minute, hour, day,
-#' week, month, quarter, or year.
+#' week, month, "bimonth", quarter, half-year, or year.
```

Comment:
I quoted "bimonth", because I don't think there is a proper term for "two-month period" and "bimonth" is just a made-up word. Same with "halfyear". It might be better though if the terms were consistent with the options.